macosx/Torrent.mm

```diff
@@ -429,6 +446,18 @@ - (NSString*)magnetLink
     return @(tr_torrentGetMagnetLink(fHandle));
 }
 
+- (NSString*)magnetLocation
+{
+    if (self.isMagnet)
+    {
+        if (tr_torrentGetDownloadDir(fHandle) != NULL)
```

Comment:
This can be combined with the condition above

Reply:
old PR